test/e2e/e2e_test.go

```diff
@@ -33,10 +33,14 @@ import (
 	// "github.com/onsi/ginkgo"
 
 	"k8s.io/component-base/version"
+	conformancetestdata "k8s.io/kubernetes/test/conformance/testdata"
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/config"
 	"k8s.io/kubernetes/test/e2e/framework/testfiles"
-	"k8s.io/kubernetes/test/e2e/generated"
+	e2etestingmanifests "k8s.io/kubernetes/test/e2e/testing-manifests"
+	e2enodetestingmanifests "k8s.io/kubernetes/test/e2e_node/testing-manifests"
```

Comment:
are packages with hyphens ok?

Reply:
I couldn't find any concrete evidence to say otherwise. I didn't want to change how the manifests are stored and the directories.

Reply:
`e2e_node/testing-manifests` makes me hurt a little inside, but agree on minimizing unnecessary movement

Reply:
> e2e_node/testing-manifests makes me hurt a little inside

Indeed. The mixed directory naming looks weird.